gqueries/output_elements/output_series/sankey_energy_overview/sankey_0_to_2_ammonia_to_industry.gql

```diff
@@ -1,2 +1,6 @@
-- query = Q(final_demand_of_ammonia_in_industry_energetic)
+- query = 
+    SUM(
+      Q(final_demand_of_ammonia_in_industry_energetic),
+      Q(final_demand_of_ammonia_in_industry_non_energetic)
```

Comment:
I think we show non_energetic demand as a dedicated flow called "feedstock" in the Sankey chart. Is it correct to add the flow here?

Reply:
You're right! The non energetic flow shouldn't be in this query so I'll remove this.